docs/reference/operator-config.md

```diff
@@ -0,0 +1,7 @@
+---
+sidebar_position: 2
```

Comment:
```suggestion
```
let's actually just remove these for all docs in the `reference` category, that way it'll just sort alphabetically

Reply:
oh it just hit me what this means. I'll fix it.